### DISCUSSION TITLE
Can't open browser within AppImage?

### DISCUSSION THREAD

**Discussion by mysteryx93** [AvaloniaUI/Avalonia#11244] [Q&A] [answered]
After publishing, I'm having an issue that I can't open the browser within an AppImage.

Strangely, the FluentAvalonia Hyperlink button does work. [So I took the code here.](https://github.com/amwx/FluentAvalonia/blob/master/src/FluentAvalonia/UI/Controls/Button/HyperlinkButton.cs)

[Copied it here.](https://github.com/mysteryx93/HanumanInstituteApps/blob/482a08d2c1ae38244b36f48f7c13c3d06a181e22/Services/Services/ProcessService.cs#L27)

Still doesn't work. What am I missing?

**Comment by maxkatz6** [comment]
Not sure, but googling showed some hints https://github.com/AppImage/AppImageKit/issues/124#issuecomment-205486221

**Reply by mysteryx93** [reply]
Ya saw that; but the Hyperlink button DOES work within an AppImage, so...

**Comment by mysteryx93** [comment]
Curious, it's not working in Windows either?

Is there any reason why it would work fine in development but not after publish?

**Reply by mysteryx93** [reply]
OK the click simply isn't being processed after being published.

It's a trimming issue.

**Comment by mysteryx93** [comment, marked as answer]
Why could the Click possibly fail after trimming?

Hah! This needs to be set on side libraries too, not just the output project.

    <AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>

That fixes it.
